projects/golang/go/1.16/patches/0101-syscall-expose-IfInfomsg.X__ifi_pad-on-s390x.patch

```diff
@@ -0,0 +1,41 @@
+From 2e472ee2cfd536a2f62c2c66be47dc831f524beb Mon Sep 17 00:00:00 2001
```

Comment:
Remind me why this one has a different structure than the other patches?

Reply:
yeah these are patches that were included by upstream in the RPM; they have a different format than our patches, which I prefaced with the version branch we used as the source. happy to change the format on these guys if you have suggestions

Reply:
Fine with me for now until we learn otherwise.